browser/components/newtab/tests/xpcshell/test_RemoteNewTabLocation.js

```diff
@@ -0,0 +1,39 @@
+/* globals Assert, RemoteNewTabLocation, Services */
+"use strict";
+
+Components.utils.import("resource:///modules/RemoteNewTabLocation.jsm");
+Components.utils.import("resource://gre/modules/Services.jsm");
+Components.utils.importGlobalProperties(["URL"]);
```

Comment:
Nit: You probably don't need to import this here. It should be in this environment.